From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@electronjs.org>
Date: Tue, 1 Jul 2025 15:04:17 -0400
Subject: build: update siso-chromium image

Make sure that the siso-chromium image matches what we are actually
using for siso.

diff --git a/buildtools/reclient_cfgs/linux/chromium-browser-clang/rewrapper_linux.cfg b/buildtools/reclient_cfgs/linux/chromium-browser-clang/rewrapper_linux.cfg
index 6f6e1ca9a0c1385b94aa3b46be5a0a143fc57835..b38932dfc0105c224de0ad762d4468140b4760dc 100644
--- a/buildtools/reclient_cfgs/linux/chromium-browser-clang/rewrapper_linux.cfg
+++ b/buildtools/reclient_cfgs/linux/chromium-browser-clang/rewrapper_linux.cfg
@@ -1,4 +1,4 @@
-platform=container-image=docker://gcr.io/chops-public-images-prod/rbe/siso-chromium/linux@sha256:26de99218a1a8b527d4840490bcbf1690ee0b55c84316300b60776e6b3a03fe1,label:action_default=1
+platform=container-image=docker://gcr.io/chops-public-images-prod/rbe/siso-chromium/linux@sha256:ef35d347f4a4a2d32b76fd908e66e96f59bf8ba7379fd5626548244c45343b2b,label:action_default=1
 server_address=unix:///tmp/reproxy.sock
 labels=type=compile,compiler=clang,lang=cpp
 exec_strategy=racing
